pulpanalytics/migrations/0007_alter_system_postgresql_version.py

```diff
@@ -0,0 +1,17 @@
+# Generated by Django 4.1.7 on 2023-03-08 14:33
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("pulpanalytics", "0006_system_custom_access_policies_system_custom_roles_and_more"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="system",
+            name="postgresql_version",
+            field=models.PositiveIntegerField(default=0),
```

Comment:
not also null? You decide, but I figured do it the same as the others?

Reply:
0 had a meaning for this field already.